src/features/newTask.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { defaultEqualityCheck } from "reselect";
+
+
+export const newTaskSlice = createSlice({
+    name: "newTask",
+    initialState: {value: ""},
+    reducers: {
+        newesTask: (state, action) => {
```

Comment:
I believe this naming can be improved. Try to find a meaningful name to use instead of `newesTask`